app/dashboard/notifications.py

```diff
@@ -254,6 +255,49 @@ def maybe_market_to_user_slack(bounty, event_name):
     return sent
 
 
+def maybe_market_to_user_discord(bounty, event_name):
+    """Send a Discord message to the user's discord channel for the specified Bounty.
+
+    Args:
+        bounty (dashboard.models.Bounty): The Bounty to be marketed.
+        event_name (str): The name of the event.
+
+    Returns:
+        bool: Whether or not the Discord notification was sent successfully.
+
+    """
+    from dashboard.models import Profile
+    if bounty.get_natural_value() < 0.0001:
+        return False
+    if bounty.network != settings.ENABLE_NOTIFICATIONS_ON_NETWORK:
```

Comment:
While testing on local, I realised that this env variable needs to be set to `rinkeby` for the message to be sent out for both the discord and slack integrations. Is this necessary or was it originally meant for gitcoinbot notifications only?

If it should be left in, I suggest adding a note in the readme for running locally (or in the FAQ) that states this prerequisite explicitly when testing slack/discord/other notifications.

Reply:
> Is this necessary or was it originally meant for gitcoinbot notifications only?

yeah it was meant to make sure that people didnt accidently make a bunch of github comments for no reason 

@mbeacom has since simplified the logic i believe